### ISSUE TITLE
Number in country code `CI` became invalid as of `0.6.51`

### ISSUE THREAD

**Issue by theblang** [daddyz/phonelib#230]
I created a sample number based off the number of one of our users. Running `Phonelib.valid?('+22501234567')` returns true in `0.6.50`, but false in `0.6.51`. I glanced at the `libphonenumber` changelog, perhaps [8.12.23 or 8.12.22](https://github.com/google/libphonenumber/blob/master/release_notes.txt#L21) is to blame?

**Comment by kbroderick** [comment]
I'm seeing Puerto Rican phone numbers that had been valid under 0.6.46 showing invalid under 0.6.51. We use the AR validation, i.e. 
```ruby
validates :contact_phone, phone: { allow_blank: true, countries: APP_CONFIG['countries'] })
```
where APP_CONFIG['countries'] returns ["us", "ca", "pr", "mx"].

Phonelib.valid_in_country? "787-241-8453", "pr" returns true, but an AR record d87-241i-8453" returns an empty set when p.valid_countries is called. p.valid_in_country? "pr" returns true as expected; "usa" returns false (which seems to be the expected result in that case).

**Comment by daddyz** [comment]
@theblang Yes, I always advice to check [here](https://rawgit.com/googlei18n/libphonenumber/master/javascript/i18n/phonenumbers/demo-compiled.html). Check ```add_additional_regex``` documentation, it may help solving the issue. Or you can open an issue for libphonenumber [here](https://issuetracker.google.com/issues/new?component=192347&template=829703&pli=1)

**Comment by theblang** [comment]
Oh funny, [looks like](https://support.signal.org/hc/en-us/articles/360056572751-Ivory-Coast-phone-number-renumbering) phone numbers were restructured there to have an additional "two digit provider code".
